fast64_internal/f3d/f3d_writer.py

```diff
@@ -140,6 +140,29 @@ def getInfoDict(obj):
     return infoDict
 
 
+def getSTUVRepeats(tex_prop: "TextureProperty") :
+    SShift, TShift = 2 ** tex_prop.S.shift, 2 ** tex_prop.T.shift
+    sMirrorScale = 2 if tex_prop.S.mirror else 1
+    tMirrorScale = 2 if tex_prop.T.mirror else 1
+    return (SShift * sMirrorScale, TShift * tMirrorScale)
+
+
+def getUVInterval(f3dMat) -> tuple[int, int]:
```

Comment:
I don't think this behaves as expected if e.g. tex0 has shift=0 mirror=True and tex1 has shift=1 mirror=False